scripts/build_dpdk.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# -DHAVE_TC_FLOWER -DHAVE_TC_VLAN_ID
+DPDK_CFLAGS="-fPIC"
+
+echo dpdk source dir: $1
+cd $1
+
+EXTRA_CFLAGS=$DPDK_CFLAGS
+
+make EXTRA_CFLAGS="$DPDK_CFLAGS"
```

Comment:
Be careful here, this script fails on make error only because make is the last command.
You may add a `set -e` on line 2